akka-http-core/src/main/scala/akka/http/model/HttpEntity.scala

```diff
@@ -160,11 +160,11 @@ object HttpEntity {
     override def toStrict(timeout: FiniteDuration)(implicit fm: FlowMaterializer) =
```

Comment:
Well, the member is at same time something we accept from the user as well as something that we supply. So we have conflicting goals of accepting `Any` but supplying `Unit`.
I'll make the change to `Any` here and in all other similar cases.